package.json

```diff
@@ -93,6 +93,7 @@
     "jsonschema": "^1.1.0",
     "minimatch": "^3.0.4",
     "parse5": "^4.0.0",
+    "path-is-inside": "^1.0.2",
```

Comment:
Cool, zero dependencies and written by Domenic

Reply:
Yup!